lms/djangoapps/teams/__init__.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+
+
+def is_feature_enabled(course):
+    """
+    Returns True if the teams feature is enabled.
+    """
+    return settings.FEATURES.get('ENABLE_TEAMS', False) and course.teams_enabled
```

Comment:
Need a newline (check the pylint/pep8 errors)-- grab one of us to show you how to navigate to the report if you haven't done it before.